docs/reference/migration/migrate_6_0/mappings.asciidoc

```diff
@@ -1,6 +1,17 @@
 [[breaking_60_mappings_changes]]
 === Mapping changes
 
+==== The `_all` meta field is now disabled by default
+
+On new mappings, the `_all` meta field that contains a copy of the text from
+each field is now disabled by default. The `query_string` and
+`simple_query_string` queries that previously used `_all` to search will now
+check if `_all` is enabled/disabled and switch to executing the query across all
+fields if `_all` is disabled. `_all` may be re-enabled by setting `enabled`:
+`true` in the mapping configuration for `_all`. If `_all` is enabled on an
+existing index data that was previously indexed will need to be reindexed in
+order to populate the `_all` field.
+
```

Comment:
We do disallow this for 5.0 indices, see `UpdateMappingOnClusterIT` which tests this